draft-jhoyla-req-mtls-flag.md

```diff
@@ -144,6 +144,49 @@ terminate the connection with a fatal `illegal_parameter` alert.
 
 # Security Considerations
 
+## Authentication Properties
+
+TLS client authentication is bound to a particular TLS session. This means that
+a malicious server cannot manipulate a client in such a way that they can
+impersonate said client to any other server. This is due to the matching
+sessions property of TLS.
+
+This differs sharply from e.g. HTTP Signatures, where the receiving server can
+relay a signature header to another server. In HTTP Signatures this is
+intentional, as it enables the use of proxies.
+
+This is because they authenticate different things. TLS client authentication
+identifies the actor you are talking to directly, whereas HTTP signatures
+identifies the actor that triggered some request, but not necessarily the actor
+you are in direct contact with.
+
+We describe the difference between these two properties as "principal
+authentication" and "agent authentication". Principals are the initiators of a
+particular sequence of actions. It is expected for their authentication
+information to be proxied from agent to agent as the relevant actions play out.
+For example a crawler might request a webpage from a CDN, and the CDN may relay
+the crawler's authentication, in the form of a `Signature` header, to the origin
+server. Agents, on the other hand, are the immediate participants in a protocol.
+Each agent in a chain authenticates itself only to its immediate neighbours.
+
+The upside of agent authentication is that it is session-matching, i.e. it
+cannot be replayed. However that authentication cannot be passed on. The upside
+of principal authentication is that it can be passed down an entire chain of
+agents, and each agent can verify it independently. However, it suffers from
```

Comment:
"chain of agents": see https://github.com/jhoyla/draft-jhoyla-req-mtls-flag/pull/19#discussion_r2150017380